website/docs/getting-started/setup/upgrade-to-business-edition/upgrade-k8s-to-business-edition.mdx

```diff
@@ -0,0 +1,21 @@
+# 
```

Comment:
Please move this to /getting-started/setup/upgrade-to-business-edition/kubernetes